### PR TITLE
Add it.dottorblaster.cauldron

### PR DESCRIPTION
**Pull request by dottorblaster**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly. `Cauldron is a GNOME desktop application written in Rust that allows you to read articles stored in your Pocket account`
- [X] The domain used for the application ID is [controlled by the application developer(s)][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read and followed all the [Submission requirements][reqs] and the [Submission guide][reqs2].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor to the project. If not, I contacted upstream developers about this submission.

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[reqs2]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: it.dottorblaster.cauldron.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        "org.freedesktop.Sdk.Extension.llvm18"
+    ],
+    "command": "cauldron",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--env=RUST_LOG=cauldron=debug",
```
Comment (hfiguiere) [inline it.dottorblaster.cauldron.json:17]:
```suggestion
```

**Comment by hfiguiere** [comment]
bot, build it.dottorblaster.cauldron

**Comment by bbhtt** [comment]
clicking this icon twice while not logged in causes it to panic and crash

image


Otherwise seems fine.

**Pull request closed without merging**